generators/generator-base.js

```diff
@@ -199,6 +304,23 @@ module.exports = class extends PrivateBase {
         this.needleApi.clientI18n.addEntityTranslationKey(key, value, language);
     }
 
+    /**
+     * Add a new menu entry in the "global.json" translations.
+     *
+     * @param {string} key - Key for the entity name
+     * @param {string} value - Default translated value
+     * @param {string} language - The language to which this translation should be added
+     */
+    addMenuTranslationKey(key, value, language, entityModule) {
+        if (entityModule === undefined || entityModule === 'entities') {
+            this.addEntityTranslationKey(key, value, language);
+        } else if (entityModule === 'admin') {
+            this.addAdminElementTranslationKey(key, value, language);
+        } else {
+            this.log(`Entry ${key} not added to translation on module ${entityModule}`);
```

Comment:
maybe adding a little bit more details here about why the key cannot be added? like 'the entity modyle isn't recognized' or something